Product/Production/Services/CORE_X12DocumentSubmissionCore/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/inbound/PassthroughInboundCORE_X12DSRealTime.java

```diff
@@ -42,7 +43,7 @@
      * Default constructor.
      */
```

Comment:
Development in progress this is interim checking. The last PR is in place now and you should be able to see both Inbound and Outbound.